.github/workflows/sandbox-creation.yml

```
@@ -0,0 +1,134 @@
+          if [ $AGE -ge $MAX_AGE ]; then
+            echo "Token older than 6d23h, rotate_needed=true"
+            echo "rotate_needed=true" >> $GITHUB_OUTPUT
+          else
+            echo "Token still valid, rotate_needed=false"
+            echo "rotate_needed=false" >> $GITHUB_OUTPUT
+          fi
+
+  TriggerRotation:
+    runs-on: ubuntu-latest
+    needs: [CheckTokenInSecretsManager]
+    if: needs.CheckTokenInSecretsManager.outputs.rotate_needed == 'True'
```

actionlint (1.7.4)

property "rotate_needed" is not defined in object type {}

(expression)